phylogenetic/rules/export.smk

```diff
@@ -32,9 +32,9 @@ OUTPUTS:
 
 rule remove_time:
     input:
-        "results/{build_name}/branch_lengths.json",
+        build_dir + "/{build_name}/branch_lengths.json",
```

Comment:
this is probably some snakemake lore i haven't absorbed yet, but i wonder: why concatenate instead of interpolate? (e.g., `"{build_dir}/{build_name}/branch_lengths.json"`)